src/completion/completionImports.ts

```diff
@@ -0,0 +1,43 @@
+// Copyright 2023 CodeMaker AI Inc. All rights reserved.
+
+import * as vscode from 'vscode';
+
+// TODO add more import statements for different languages
+const importStatements = [
+    /Add import.*/,
```

Comment:
Curious, none of that action has some form of distinctive identifier like type of suggestion or implementation type? Is the name the only way to identify them?

Reply:
Will this also work with different translations? Like Spanish, German?

Reply:
Today, they all have the same kind as `quickFix`, there is no other identifier we can use.

I tested with Chinese Language package, it shows english as well. But it worth to check others as well.